server/datastore/mysql/software_titles.go

```diff
@@ -151,13 +151,26 @@ func (ds *Datastore) ListSoftwareTitles(
 			if title.PackageVersion != nil {
 				version = *title.PackageVersion
 			}
+
 			title.SoftwarePackage = &fleet.SoftwarePackageOrApp{
 				Name:               *title.PackageName,
 				Version:            version,
 				SelfService:        title.PackageSelfService,
 				PackageURL:         title.PackageURL,
 				InstallDuringSetup: title.PackageInstallDuringSetup,
 			}
+
+			policies, err := ds.GetPoliciesBySoftwareTitleID(ctx, title.ID, opt.TeamID)
```

Comment:
This creates an "N+1" queries situation, where we run an extra query for each software title to be returned. This is not ideal from a performance perspective, especially since an API user could request a large number of titles per page. Again, depends on urgency of merging this feature, but we should probably have a batch-get-policies for a list of software title IDs so it's only one additional query (maybe creating a ticker if we can't address now).